Bio/PDB/DSSP.py

```diff
@@ -412,16 +412,21 @@ def __init__(
             Sander & Rost (1994), or Wilke: Tien et al. 2013, as string
             Sander/Wilke/Miller. Defaults to Sander.
         file_type: string
-            File type switch, either PDB or DSSP with PDB as default.
+            File type switch: either PDB, MMCIF or DSSP. Inferred from the
+            file extension by default.
 
         """
         self.residue_max_acc = residue_max_acc[acc_array]
 
         # create DSSP dictionary
+        if file_type == "":
+            file_type = os.path.splitext(in_file)[1][1:]
```

Comment:
~~This is a very nitpicky comment but sometimes I name files `1abc.refined.pdb`, so this would fail to get the right extension. Since I'm probably not the only one, could I bother you to change this line to `os.path.splitext(in_file)[-1][1:]`?~~

Nevermind, I just checked and `splitext` cuts by the last period!